File: parent-337/pinyougou-web-manager/src/main/webapp/js/service/sellerService.js
app.service('sellerService', function ($http) {

    this.add = function (entity) {
        return $http.post('../seller/add.do', entity);
    };

    this.search = function (pageNum, pageSize, searchEntity) {
        return $http.post('../seller/search.do?pageNum=' + pageNum + '&pageSize=' + pageSize, searchEntity);
    };

    this.findOne = function (sellerId) {
        return $http.get('../seller/findOne.do?sellerId=' + sellerId);
    };

    this.updateStatus = function (sellerId, status) {
        return $http.get('../seller/updateStatus.do?sellerId=' + sellerId + '&status=' + status);
    }
});

File: parent-337/pinyougou-interface/src/main/java/cn/itcast/core/service/SellerService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.seller.Seller;
import entity.PageResult;

public interface SellerService {

    /**
     * 商家入驻
     * @param seller
     */
    void add(Seller seller);

    /**
     * 分页+查询
     * @param pageNum
     * @param pageSize
     * @param seller
     * @return
     */
    PageResult search(Integer pageNum, Integer pageSize, Seller seller);

    /**
     * 查询商品详情
     * @param sellerId
     * @return
     */
    Seller findOne(String sellerId);

    /**
     * 修改商品状态
     * @param sellerId
     * @param status
     */
    void updateStatus(String sellerId, String status);
}
